x-pack/plugins/siem/public/lib/track_usage/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
```

Comment:
I took away the `@ts-ignore` and didn't run into any issues with type checking. I ran the command:

```ts
node scripts/type_check.js
```

And didn't run into problems removing the ts-ignore

Reply:
The error showed up in `yarn run build`, which is executed on Jenkins.

Reply:
Ok, I replicated this through the above command and think I found a fix.

It looks like what you want is this:

```ts
import { trackUiMetric } from 'src/legacy/core_plugins/ui_metric/public';
```

That passes both the type checker and the build because of this ts config compiler option:
https://github.com/elastic/kibana/blob/master/tsconfig.json#L9

Ran these two below and it looked to work with both.
```ts
node scripts/type_check.js
yarn run build
```